apps.toml

```diff
@@ -199,6 +203,7 @@ level = 7
 state = "working"
 subtags = [ "books" ]
 url = "https://github.com/YunoHost-Apps/bicbucstriim_ynh"
+antifeatures = ["deprecated-software"]
```

Comment:
new repo seems to be maintained ? cf https://github.com/YunoHost-Apps/BicBucStriim_ynh/pull/17

Reply:
```suggestion
url = "https://github.com/YunoHost-Apps/bicbucstriim_ynh"
```

Reply:
Yes indeed, might have been an automatic flag in this pr.